db/migrate/20230117190802_create_workflow_migrations.rb

```diff
@@ -0,0 +1,26 @@
+class CreateWorkflowMigrations < ActiveRecord::Migration[6.1]
+  def change
+    create_table :workflows do |t|
+      t.references :ems,           :type => :bigint, :index => true, :references => :ext_management_system
+      t.references :tenant,        :type => :bigint, :index => true
+      t.references :git_reference, :type => :bigint, :index => true
+      t.string :type, :index => true
+      t.jsonb :payload, :default => {}
+
+      t.timestamps
+    end
+
+    create_table :workflow_instances do |t|
+      t.references :ems,      :type => :bigint, :index => true, :references => :ext_management_system
+      t.references :workflow, :type => :bigint, :index => true
+      t.references :tenant
+      t.string :type, :index => true
+      t.string :userid
+      t.string :state
```

Comment:
Is the actual current step?  or something else like "running"?  How is this different from status? (I'm ok with the names, but state/status has always been confusing...wonder if "step" might be a clearer differentiator).

Reply:
Oh yeah `state` is confusing, this is not the current state this matches the miq_request_task state/status so state would be "pending", "running", "finished" and status is "error", "warning"

We could probably collapse into a single "status" though where "queued", "running" are the "active" statuses and once finished "success", "warning", "error" are the "completed" statuses